docs/includes/using-preview-apis-host.md

```diff
@@ -0,0 +1,7 @@
+> [!NOTE]
+> Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
+>
+> To use preview APIs:
+>
+> - You must reference the **beta** library on the CDN (https://appsforoffice.microsoft.com/lib/beta/hosted/office.js).
+> - You may need to join the [Office Insider program](https://products.office.com/office-insider) for access to more recent Office builds.
```

Comment:
I should have thought of this last pass, but should we add a line about type information? Like, 
"Reference `@types/office-js-preview` for Intellisense type information and TypeScript compilation."

Reply:
I'll do that in a subsequent change. I think we should include similar info for released requirement sets if we don't already, for consistency.